go/vt/vtctl/vtctl.go

```diff
@@ -1576,6 +1578,26 @@ func commandCreateKeyspace(ctx context.Context, wr *wrangler.Wrangler, subFlags
 		wr.Logger().Infof("keyspace %v already exists (ignoring error with -force)", keyspace)
 		err = nil
 	}
+
+	vschema, err := wr.TopoServer().GetVSchema(ctx, keyspace)
+	if *allowEmptyVSchema == false && (vschema == nil || err != nil) {
```

Comment:
The err != nil check here seems problematic -- if for some reason there is a vschema but there was an error fetching it, then this path would stomp on the existing one. Probably not a huge deal, but I'd rather us explicitly check for the err indication that the vschema doesn't exist and not blanket fail on all errors.